.chloggen/ingest_tls_openshift.yaml

```diff
@@ -0,0 +1,28 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use TLS via OpenShift service annotation when gateway/multitenancy is disabled
+
+# One or more tracking issues related to the change
+issues: [963]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
```

Comment:
Could we document here how clients (e.g. OTEL col) can configure TLS? e.g. use `service.beta.openshift.io/inject-cabundle` to provision the cert into a config map or use `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt` ?

Reply:
Sure